rlbottraining/common_graders/rl_graders.py

```diff
@@ -16,27 +16,63 @@
 from rlbottraining.common_graders.compound_grader import CompoundGrader
 from rlbottraining.common_graders.timeout import FailOnTimeout, PassOnTimeout
 from rlbottraining.common_graders.goal_grader import PassOnGoalForAllyTeam
+from rlbot.training.training import Pass, Fail, Grade
 
 
 class RocketLeagueStrikerGrader(CompoundGrader):
     """
     A Grader which aims to match the striker training.
     """
 
-    def __init__(self, timeout_seconds=4.0, ally_team=0):
+    def __init__(self, timeout_seconds=4.0, ally_team=0, timeout_override=False, ground_override=False):
```

Comment:
Please add documentation about what the overrides do.
Will users of this class care about this configurability?

Reply:
Probably users will not care, I cant think of a good use case beyond testing the grader.